UniversalDashboard.Forge.psm1

```diff
@@ -204,30 +212,19 @@ function Set-SquirrelConfig {
     param(
         [Parameter(Mandatory)]
         $ConfigPath,
-
         $IconUrl,
-
         $SetupIcon,
-
         $LoadingGif
     )
 
-    $Content = Get-Content -Path $ConfigPath -Raw | ConvertFrom-Json
-    $SquirrelConfig = $Content.config.forge.makers | Where-Object {$_.name -like '*squirrel'}
-
-    if ($IconUrl) {
-        Write-Verbose "Setting SquirrelConfig IconUrl: $IconUrl"
-        $SquirrelConfig.config | Add-Member -MemberType NoteProperty -Name 'iconUrl' -Value $IconUrl
-    }
-
-    if ($SetupIcon) {
-        Write-Verbose "Setting SquirrelConfig SetupIcon: $SetupIcon"
-        $SquirrelConfig.config | Add-Member -MemberType NoteProperty -Name 'setupIcon' -Value $SetupIcon
-    }
+    $content = Get-Content -Path $ConfigPath -Raw | ConvertFrom-Json
+    $squirrelConfig = $Content.Config.Forge.Makers.Where({$_.Name -like '*squirrel'}).Config
+    $keys = $PSBoundParameters.Keys.Where({$_ -ne 'ConfigPath'})
 
-    if ($LoadingGif) {
-        Write-Verbose "Setting SquirrelConfig LoadingGif: $LoadingGif"
-        $SquirrelConfig.config | Add-Member -MemberType NoteProperty -Name 'loadingGif' -Value $LoadingGif
+    foreach ($parameter in $keys) {
+        Write-Verbose ('Setting SquirrelConfig {0}: {1}' -f $parameter, $PSBoundParameters[$parameter])
+        $name = $parameter -replace '^\w', $parameter.Substring(0, 1).ToLower()
+        $squirrelConfig | Add-Member -MemberType NoteProperty -Name $name -Value $PSBoundParameters[$parameter]
```

Comment:
As more properties are supported for the squirrel config this function would potentially get huge and be harder to maintain. Using `$PSBoundParameters` all that would then be needed is to add the new Parameter and that's it.